src/models/SinglePoll.ts

```diff
@@ -0,0 +1,66 @@
+import TwitterInfo from './TwitterInfo'
+import VotesInfo from './VotesInfo'
+
+export default class SinglePoll {
```

Comment:
Don't know for sure, but if this effective directly implements `ISinglePoll`, do this:
```suggestion
export default class SinglePoll implements ISinglePoll {
```